zenodo/modules/records/data/objecttypes.json

```
@@ -264,7 +264,7 @@
     "en": "Conference papers"
   },
   "schema.org": "https://schema.org/ScholarlyArticle",
-  "datacite": {"general": "Text", "type": "Conference paper"},
+  "datacite": {"general": "Conference paper"},
```

```suggestion
  "datacite": {"general": "ConferencePaper"},
```

We need the exact values from the schema, since this will be used when sending over metadata.